fluent-bit-daemonset-kafka-rest.yml

```diff
@@ -4,14 +4,14 @@ metadata:
   name: fluent-bit
   namespace: kube-system
   labels:
-    k8s-app: fluent-bit-logging
+    app.kubernetes.io/name: fluent-bit-logging
     version: v1
```

Comment:
Shouldn't this also be updated?

Reply:
Good catch, thanks.
I haven't seen a version label used very much in k8s. 
Generally the image tag is sufficient or CD tool injects suitable label.
I'd be tempted to dump it. WDYT?

Reply:
I don't see much of a use for it either in this case. It's probably more useful when using gitops or for running multiple versions of mysql or something.

But if version is to be kept, then it should use the recommend label.

Reply:
I disagree, and think it should be kept. We (@moovweb) can help to create a Helm chart for this repo based on our rollout of this, and the `app.kubernetes.io/version` label would be used.

Reply:
Ok. All done (did on mobile hence number of commits).